drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ReferralPosting < Types::BaseObject
+    field :id, ID, null: false
+
+    # Fields that come from Referral
+    field :referral_identifier, ID, null: true
+    field :referral_date, GraphQL::Types::ISO8601Date, null: false
+    field :referred_by, String, null: false
+    field :referral_notes, String, null: true
+    field :chronic, Boolean, null: true
+    field :score, Integer, null: true
+    field :needs_wheelchair_accessible_unit, Boolean, null: true
+
+    # Fields that come from ReferralHouseholdMembers
+    field :hoh_name, String, null: false
+    field :household_size, Integer, null: false
+
+    # Fields that come from Posting
+    field :resource_coordinator_notes, String, null: true
+    field :posting_identifier, ID, null: false, method: :identifier
+    field :assigned_date, GraphQL::Types::ISO8601Date, null: false, method: :created_at
+    field :referral_request, HmisSchema::ReferralRequest, null: true
+    field :status, HmisSchema::Enums::ReferralPostingStatus, null: false
+    field :status_updated_at, GraphQL::Types::ISO8601Date, null:true
+    field :status_updated_by, String, null:true
+    field :status_note, String, null: true
+    field :status_note_updated_at, GraphQL::Types::ISO8601Date, null:true
```

Comment:
Rubocop should be complaining about `null:true`, I think? Do you have the overcommit hooks set up?

Reply:
I don't use overcommit but rubocop isn't complaining about that file when I invoke it manually.  I pulled out the `null: true` here.

In the past I've had to add rubocop-graphql to lint graphql. Is there some other way we are doing it on this project?

Reply:
Huh, odd. I just meant the inconsistency between `null:true` and `null: true` in the file.
When I make a similar change, I get an error `[Correctable] Layout/SpaceAfterColon: Space missing after colon.`.

I typically just use my vscode to autocorrect errors.

When you run it manually, are you sure you're using `.rubocop.yml`?
There's nothing graphql specific here, just ruby spacing.

Reply:
Oh sorry I misunderstood. Yeah, rubocop should have flagged that missing space. I see the linter errors in vscode on most files and it seems to be respecting .rubocop.yml. I'm not sure why they weren't showing for me here